auth/build.gradle.kts

```diff
@@ -13,6 +13,10 @@ android {
         disable("UnknownNullness")  // TODO fix in future PR
         disable("TypographyQuotes") // Straight versus directional quotes
         disable("DuplicateStrings")
+        disable("MissingTranslation") // TODO add translations
```

Comment:
Have these gone in?

Reply:
Looks like they're ready now, I'll go get them